src/usersAPI/users-controller.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+// Imports:
+const fs = require('fs');
+const path = require('path');
+
+// Path:
+const usersDataBasePath = path.join(__dirname, '..', 'data', 'users.json');
+
+// Array for tests only; user.json file could store all the data permanently.
+let usersArr = [];
+
+// Function to get the data from the array/json/db:
+const getUsersFromDB = () => {
+  const data = fs.readFileSync(usersDataBasePath).toString();
+
+  // TODO:
+  // Return this from this func but NOT usersArr to use .json as DB.
+  JSON.parse(data);
+
+  // Not this:
+  return usersArr;
+};
```

Comment:
same here about additional functionality
let's create separate service and move it there